src/utils.js

```diff
@@ -0,0 +1,90 @@
+var _ = require('lodash');
```

Comment:
This whole file will need to be wrapped in
```javascript
go.utils = function () {
...
  return {
    ...
  };
}();
```
the way `init.js` and `app.js` do.